content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -34,7 +34,7 @@ <h3>Kubernetes Deployments</h3>
                 master schedules mentioned application instances onto individual Nodes in the cluster.
                 </p>
 
-                <p>Once the application instances are created, a Kubernetes Deployment Controller continuously monitors those instances. If the Node hosting an instance goes down or is deleted, the Deployment controller replaces it. <b>This provides a self-healing mechanism to address machine failure or maintenance.</b></p>
+                <p>Once the application instances are created, a Kubernetes Deployment Controller continuously monitors those instances. If the Node hosting an instance goes down or is deleted, the Deployment controller replaces the instance on another Node in the cluster. <b>This provides a self-healing mechanism to address machine failure or maintenance.</b></p>
```

Comment:
```suggestion
                <p>Once the application Pods are created, a Kubernetes Deployment Controller continuously monitors those Pods. If the Node hosting an instance goes down or is deleted, the Deployment controller replaces the Pod on another Node in the cluster. <b>This provides a self-healing mechanism to address machine failure or maintenance.</b></p>
```
Unless the word `instance` makes more sense to a user, we should use the terminology that Kubernetes has for these concepts.

Reply:
The rest of the document is using `instance` and for consistency, I used the word`instance` or we need to re-write the entire doc replacing `application instance` with `pod`?

Reply:
@daminisatya good question, I suppose since this is in the "Kubernetes Basics" section, overloading folks with terminology might not be ideal. Whichever one you'd like to use as long as it's consistent is good enough for me!